concordium-node/src/bin/bootstrapper.rs

```diff
@@ -44,12 +46,22 @@ fn main() -> anyhow::Result<()> {
         conf.common.id,
         &conf,
         PeerType::Bootstrapper,
-        stats_export_service,
+        stats_export_service.clone(),
         regenesis_arc,
     )
     .context("Failed to create the network node.")?;
 
     start_push_gateway(&conf.prometheus, &node.stats, node.id());
+
+    if let Some(plp) = conf.prometheus.prometheus_listen_port {
+        let (sender, _) = tokio::sync::broadcast::channel(1);
```

Comment:
DOcument here that we don't care about graceful shutdown so we ignore the receiver.